modules/virt-hot-plugging-bridge-network-interface-cli.adoc

```diff
@@ -22,21 +23,46 @@ Hot plug a bridge network interface to a virtual machine (VM) while the VM is ru
 $ virtctl start <vm_name>
 ----
 
-. Use the following command to hot plug a new network interface to the running VM. The `virtctl addinterface` command adds the new network interface to the VM and virtual machine instance (VMI) specification but does not attach it to the running VM.
+. Add the new network interface to the running VM by editing the VM specification. This adds the new network interface to the VM and virtual machine instance (VMI) specification but does not attach it to the running VM.
 +
-[source,terminal]
+.Example VM configuration
+[source,yaml]
 ----
-$ virtctl addinterface <vm_name> --network-attachment-definition-name <net_attach_def_name> --name <interface_name>
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+template:
+  spec:
+    domain:
+      devices:
+        interfaces:
+        - name: defaultnetwork
+          masquerade: {}
+        # new interface
+        - name: <secondary_nic> # <1>
+          bridge: {}
+    networks:
+    - name: defaultnetwork
+      pod: {}
+    # new network
+    - name: <secondary_nic> # <2>
+      multus:
+        networkName: <nad_name> # <3>
+# ...
 ----
-+
-where:
-
-<vm_name>:: Specifies the name of the `VirtualMachine` object.
-<net_attach_def_name>::  Specifies the name of `NetworkAttachmentDefinition` object.
-<interface_name>:: Specifies the name of the new network interface.
+<1> Specifies the name of the new network interface.
+<2> Specifies the name of the network. This must be the same as the `name` of the new network interface that you defined in the `template.spec.domain.devices.interfaces` list. 
+<3> Specifies the name of the `NetworkAttachmentDefinition` object. 
 
+. Apply the VM configuration by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f vm-fedora.yaml
+----
```

Comment:
I guess this should be removed, now that the doc suggest using `oc edit`

Reply:
I got it wrong, it doesn't specify using 'oc edit'

Reply:
> I got it wrong, it doesn't specify using 'oc edit'

But it should

Reply:
I guess the reason is the fact we dont know weather VM created with `create` or `apply` command, right?